src/modifier_gene.hpp

```diff
@@ -22,6 +22,16 @@ class ModifierGene : public Gene {
 private:
 	float _modifierValue;
 	String _geneName;
+
+	String _modifierGeneTypes[7] = {
+		"ACTIVATION_THRESHOLD",
+		"STRENGTH",
+		"CONVERSION_RATE",
+		"GROWTH_CAP",
+		"ACTIVATION_RESOURCE",
+		"THRESHOLD_TYPE",
+		"N_SUBSTRUCTURES"
+	};
```

Comment:
Currently _modifierGeneTypes isn't being used anywhere. What's the goal?

Reply:
I believe this was added by Myles a while ago. it is redundant right now so we can probably just get rid of it or fully implement it through the hpp file so it can be used in the randomModifierGene function